### PR TITLE
Bug/responsive graphs 1

### PR DESCRIPTION
**Pull request by Ethan-Edmond**
# Description

[Explanation Video](https://www.loom.com/share/72714fe2247b46029ea1a162c851aa13)
This PR makes the pie graph responsive, removes pie related code from index.css, corrects an error in the pie graph legend and makes the graph container do the alignment instead of each individual graph.

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [X] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [X] `npm test`

# Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

### CODE REVIEW

**Review by austin-carman — CHANGES_REQUESTED** [review]
Overall, looks good. Just wanted to check if there is a reason the graph previously did not include officer presence and only included incidents of force? Perhaps that's how the stakeholder wanted the graph? Once that is cleared up then I'd be ready to approve

**Comment by Ethan-Edmond** [comment]
It looks like when that was written the pie graph didn't actually work yet. Other than that, I have no idea. I guess Ill ask Paul.

**Comment by Ethan-Edmond** [comment]
It looks like Paul can't find anything either, but he says it's fine (a bit of paraphrasing).

**Review by austin-carman — APPROVED** [review]
Approved after talking to Ethan about previous change suggestion

**Review by imjeremiah — APPROVED** [review]
Great explanation video, it was easy to follow your code changes along with the site demo of the changes. Also,  know this is outside the scope of the main responsive change, but great catch seeing the legend wasn't representative of the graph data and updating it to match. In light of that and the above resolved conversation, everything looks good. No merge conflicts. Approved.

File: src/components/graphs/linegraph/LineGraph.js [hunk 1]
```diff
@@ -29,13 +31,7 @@ const LineGraph = ({ data, months }) => {
...
   return (
     <>
-      <div
-        style={{
-          backgroundColor: '#191A1A',
-          margin: '0 auto',
-          maxWidth: '1550px',
-        }}
-      >
+      <div className="line-container">
```
Comment (paulstgermain) [inline src/components/graphs/linegraph/LineGraph.js:34]: Love the separation of structure and style you bring to these components.

File: src/components/graphs/linegraph/LineGraph.less [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.line-container {
+  background: #191a1a;
+  position: relative;
+  width: 100%;
+}
```
Comment (paulstgermain) [inline src/components/graphs/linegraph/LineGraph.less:5]: Not a simple copy/paste of the old styling, I can appreciate the efforts taken to implement consistent changes across similar aspects of the app here!

File: src/components/graphs/piegraph/PieGraph.less [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.pie-container {
+  position: relative;
+  width: 100%;
+  min-height: 360px;
+}
```
Comment (paulstgermain) [inline src/components/graphs/piegraph/PieGraph.less:5]: I'm curious, considering the above code:

`const graphOptions = {
  responsive: true,
  maintainAspectRatio: false,
};`

Did you find it helped the responsiveness of this graph by implementing the `graphOptions` _and_ making the width responsive with `width: 100%`? Not a merge-blocking thing, just curious about your thoughts here!

File: src/index.css [hunk 4]
```diff
@@ -1239,43 +1217,9 @@ form label {
...
   }
-  div .pie-legend {
-    width: 350px;
-    padding: 0;
-    margin-top: 20px;
-  }
-  .pie-legend div {
-    margin: 0;
-    width: 338px;
-  }
```
Comment (paulstgermain) [inline src/index.css:1278]: Really minimizing this file by moving all of the feature-specific files - nice!

**Review by paulstgermain — APPROVED** [review]
## Video Walkthrough Feedback

This was, as always, a very thorough review of the first portion of changes you made in an effort to make all of the graphs responsive - an important aspect of any modern website! 

I still appreciate the live examples of the site and the results of your work, and you still maintain a conversational style I find helpful in potentially keeping listeners engaged - something that will definitely help you in interviews!

## Code Review

The code itself is also nice - you implement consistent styling across the board, and make sensible decisions that remind even me of the premise of 'separation of concerns'. Your dedication to steadily swapping the entire application to Less is also noted - and appreciated! 

I see no problems introduced here, and the style changes are certainly welcome - approved!

**Pull request merged**